connector-runtime/connector-runtime-spring/src/main/java/io/camunda/connector/runtime/inbound/importer/ProcessDefinitionInspector.java

```diff
@@ -93,7 +92,14 @@ public List<InboundConnectorDefinitionImpl> findInboundConnectors(
     var connectorDefinitions =
         processes.stream()
             .flatMap(process -> inspectBpmnProcess(process, processDefinition).stream())
-            .collect(Collectors.groupingBy(InboundConnectorDefinition::correlationPoint));
+            .collect(
+                Collectors.groupingBy(
+                    def ->
+                        def.processDefinitionKey()
+                            + "-"
+                            + def.bpmnProcessId()
+                            + "-"
+                            + def.elementId()));
```

Comment:
Looks like either combination of `processDefinitionKey` + `elelmentId` or `bpmnProcessId` + `elementId` should be enough? Or is it somehow different for subprocesses?

Reply:
I wasnt sure whethere different subprocesses in different definitions could have the same id?

Reply:
Will go with the former.